src/store/card.js

```diff
@@ -30,8 +30,87 @@ export default {
 		cards: [],
 	},
 	getters: {
-		cardsByStack: state => (id) => {
-			return state.cards.filter((card) => card.stackId === id).sort((a, b) => a.order - b.order)
+		cardsByStack: (state, getters, rootState) => (id) => {
+			return state.cards
+				.filter((card) => card.stackId === id)
+				.filter((card) => {
+					const { tags, users, due } = rootState.filter
+					let allTagsMatch = true
+					let allUsersMatch = true
+					// const allDuedatesMatch = true
+
+					if (tags.length > 0) {
+						tags.forEach((tag) => {
+							if (card.labels.findIndex((l) => l.id === tag) === -1) {
+								allTagsMatch = false
+							}
+						})
+						if (!allTagsMatch) {
+							return false
+						}
+					}
+
+					if (users.length > 0) {
+						users.forEach((user) => {
+							if (card.assignedUsers.findIndex((u) => u.participant.uid === user) === -1) {
+								allUsersMatch = false
+							}
+						})
+						if (!allUsersMatch) {
+							return false
+						}
+					}
+
+					if (due !== '') {
+						const datediffHour = ((new Date() - new Date(card.duedate)) / 3600000)
+						switch (due) {
+						case 'overdue':
+							if (card.overdue === 0) {
```

Comment:
Can be simplified as `return card.overdue > 0`

Reply:
no, that's not working